posts/2023-12-18-survey-launch.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: "Launching the 2023 State of Rust Survey"
+author: The Rust Survey Working Group
+description: "Share your feedback with the eighth edition of the State of Rust Survey"
```

Comment:
Change to "Share your experience using Rust in the eighth edition of the State of Rust Survey"